bouncerscript/task.py

```diff
@@ -85,11 +95,13 @@ def check_product_names_match_aliases(context):
 
     validations = []
     for alias, product_name in aliases.items():
-        if alias not in ALIASES_REGEXES.keys():
+        if alias in ALIASES_REGEXES.keys():
+            validations.append(matches(product_name, ALIASES_REGEXES[alias]))
+        elif matches_partner_regex(alias, product_name):
```

Comment:
Looks like `all([]) is True`, so we could also do `elif not matches_partner_regex(...): raise...`
This works too.

Reply:
I think we'd only end up with `all([])` if `aliases_entries` was empty, so no checks to do then. I've improved the code to avoid `append(True)` which was making me twitchy, which prompted adding test so we don't point vanilla at partner and vice versa.